docarray/typing/url/url_3d/mesh_url.py

```diff
@@ -10,6 +12,11 @@
 T = TypeVar('T', bound='Mesh3DUrl')
 
 
+class Mesh3DLoadResults(NamedTuple):
```

Comment:
I think i would prefer singular naming `Mesh3DLoadResult`. Intuitively to me it is one result that has two components